assessment.md

```diff
@@ -27,23 +27,23 @@ when creating python programs.
     and development tools
   * G = Additional development tools configured, in addition to minimum required
     dependencies and tools
-  * E = Additional development tools configured and able to provide assistance
-    and troubleshooting to other students regarding their own development
+  * E = Additional development tools configured and ability to provide assistance
+    and troubleshooting to other individuals regarding their own development
     environment
 * Running Python programs with `pyenv` and `pipenv`
-  * N = `pyenv` and `pipenv` not installed or setup
+  * N = `pyenv` and `pipenv` not installed
   * I = Unable to run programs with `pyenv` and `pipenv`
   * A = Able to run programs with `pyenv` and `pipenv`
   * G = Able to run programs with `pyenv` and `pipenv` using additional commands
     and flags as necessary
   * E = Able to run programs with `pyenv` and `pipenv` using additional commands
     and flags to streamline workflow, while being able to provide assistance and
-    troubleshooting to other students using `pyenv` and `pipenv`
+    troubleshooting to other individuals using `pyenv` and `pipenv`
```

Comment:
We should remove "being able to" since this assessment is about what has been done rather than what can be done. We should not be based on being able to help others as this would be difficult to prove and unnecessary.